src/main/java/com/jackshenorion/smarts/util/io/IOUtil.java

```diff
@@ -525,7 +525,9 @@ public static void writeAndClose(File file, byte[] data) {
 
```

Comment:
**Reporter**: FINDBUGS
**Rule**: RV_RETURN_VALUE_IGNORED_BAD_PRACTICE
**Severity**: INFO
**File**: src/main/java/com/jackshenorion/smarts/util/io/IOUtil.java L447
**Source**: com.jackshenorion.smarts.util.io.IOUtil

Method ignores exceptional return value


   <p> This method returns a value that is not checked. The return value should be checked
since it can indicate an unusual or unexpected function execution. For
example, the <code>File.delete()</code> method returns false
if the file could not be successfully deleted (rather than
throwing an Exception).
If you don't check the result, you won't notice if the method invocation
signals unexpected behavior by returning an atypical return value.
</p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1509122434>*